op-challenger/fault/utils.go

```diff
@@ -0,0 +1,24 @@
+package fault
+
+import (
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type LogBuilder struct {
+	ctx []interface{}
+}
```

Comment:
This doesn't belong as part of the fault module. I suspect needing it is a sign that we're adding too many fields to a logger which will just create noise in the logs.  If it really needs to exist it should be somewhere in op-service.